src/ipips/ipip-0001.md

```diff
@@ -0,0 +1,164 @@
+---
+date: 2022-06-10
+editors:
+  - name: Marcin Rataj
+    github: lidel
+  - name: wilkyr31d
+    github: wilkyr31d
+---
+
+# IPIP-0001: Lightweight Improvement Process for IPFS Specifications
+
+<!-- TODO: MAKE THIS FRONTMATTER FIELD -->
```

Comment:
Does it help if I take that on in the generator?

Reply:
Made a PR: https://github.com/ipfs/spec-generator/pull/6, would be great to get it merged and released!